dev/CommandBarFlyout/CommandBarFlyout.idl

```diff
@@ -13,6 +13,9 @@ unsealed runtimeclass CommandBarFlyout : Windows.UI.Xaml.Controls.Primitives.Fly
         Boolean AlwaysExpanded{ get; set; };
     }
 
+    void AddDropShadow();
```

Comment:
Is it intentional to add public APIs for this in CommandBarFlyout ?

Reply:
I'd prefer not to, but I added this so that CommandBarFlyoutCommandBar is able to tell CommandBarFlyout to add/remove its flyout presenter's shadow. Maybe there's an alternative?